gpkit/tests/t_geometric_program.py

```diff
@@ -113,44 +128,51 @@ def test_additive_constants(self):
 
     def test_zeroing(self):
         L = Variable("L")
-        k = gpkit.Variable("k", 0)
+        k = Variable("k", 0)
         gpkit.enable_signomials()
         sol = GP(1/L, [L-5*k <= 10]).solve(printing=False, solver=self.solver)
         self.assertAlmostEqual(sol(L), 10, self.ndig)
         gpkit.disable_signomials()
 
-    def test_CO(self):
+    def test_composite_objective(self):
         L = Variable("L")
         W = Variable("W")
         eqns = [L >= 1, W >= 1,
                 L*W == 10]
         obj = gpkit.composite_objective(L+W, W**-1 * L**-3, sub={L: 1, W: 1})
-        sol = GP(obj, eqns).solve(printing=False)
+        sol = GP(obj, eqns).solve(printing=False, solver=self.solver)
         a = sol["sensitivities"]["variables"]["w_{CO}"].flatten()
         b = np.array([0, 0.98809322, 0.99461408, 0.99688676, 0.99804287,
                       0.99874303, 0.99921254, 0.99954926, 0.99980255, 1])
         self.assertTrue((abs(a-b)/(a+b+1e-7) < 1e-7).all())
 
     def test_singular(self):
+        """GP with a singular A matrix"""
```

Comment:
Oh wow -- I was trying to respond to all the pylint "missing docstring" warnings. I guess docstrings for test methods are not ok then? Too bad, because I think we should really be writing them (to clarify the purpose of each test)